src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card card--image"
+      data-qa="card"
+    >
+      <div class="card__text">
+        <p class="card__text1">
```

Comment:
"text1" is not a great naming. You could probably cal it something like `card__title` or `card__product-name`.
Avoid using multiple names that differ only by a number, for example "text1" and "text2", those really are not the best approach as they indicate both texts have the same purpose, and it's not the case here.